### PR TITLE
feat: Create plugin process guide

### PR DESCRIPTION
**Pull request by vasfvitor**
#### What kind of changes does this PR include?

- New or updated content

#### Description
- Addresses #1487

Any improvements suggestions?

Thank you

### CODE REVIEW

File: src/content/docs/features/process.mdx [hunk 1]
```diff
@@ -3,12 +3,101 @@ title: Process
...
+// restarts the app
+await relaunch();
+```
+
+</TabItem>
+<TabItem label="Rust">
+
+```rust
+// exits the app with the given status code
+app.exit(0);
```
Comment (lorenzolewis) [inline src/content/docs/features/process.mdx:98]: Should this show where `app` is defined/created?

**Review by lorenzolewis — CHANGES_REQUESTED** [review]

File: src/content/docs/features/process.mdx (hunk 1 above)
Comment (vasfvitor) [reply src/content/docs/features/process.mdx:98]: I pushed a commit, for reference here is the rust code context:
```rust
#[tauri::command]
fn _exit (app: tauri::AppHandle) {
    app.exit(0);
}
```

File: src/content/docs/features/process.mdx [hunk 2]
```diff
@@ -3,12 +3,103 @@ title: Process
...
+await exit(0);
+
+// restarts the app
+await relaunch();
+```
+
+</TabItem>
+<TabItem label="Rust">
+
+Note that `app` is an instance of AppHandle `app: tauri::AppHandle`
```
Comment (lorenzolewis) [inline src/content/docs/features/process.mdx]:
```suggestion
Note that `app` is an instance of [`AppHandle`](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.AppHandle.html).
```

File: src/content/docs/features/process.mdx (hunk 1 above)
Comment (lorenzolewis) [reply src/content/docs/features/process.mdx:98]: Just did a tweak to link to the Rust docs for it.

**Pull request merged**